### PR TITLE
fix building on mac os

### PR DESCRIPTION
**Pull request by ortex**
JNI path for mac os should be `{JAVA_HOME}/include/darwin`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,7 +22,7 @@ dependencies {
 }
 
 ext {
-    os = System.getProperty("os.name").toLowerCase(Locale.ENGLISH)
+    os = getOsName()
     platform = System.getProperty("platform") ?: os
```
Comment (gavv) [inline build.gradle:26]: If I understand correctly, `os` is used only as the default value for `platform`. Thus, to make things simpler, maybe replace these two lines with just a single `platform = getPlatform()` and replace `getOsName()` with `getPlatform()`?

File: build.gradle [hunk 2]
```diff
@@ -37,7 +39,7 @@ task generateHeaders(dependsOn: 'classes') {
         classesDir.mkdirs()
 
     sourceDir.eachFileRecurse { f ->
-        if (!f.directory && f.name.substring(f.name.lastIndexOf(".")+1) == "java") {
+        if (!f.directory && f.name.endsWith("java")) {
```
Comment (gavv) [inline build.gradle:42]: Shouldn't it be `endsWith(".java")`?

File: build.gradle (hunk 1 above)
Comment (ortex) [reply build.gradle:26]: os also used in https://github.com/roc-project/roc-java/blob/master/roc_jni/build.gradle#L10

File: build.gradle (hunk 2 above)
Comment (ortex) [reply build.gradle:42]: yeah, of course

Comment (ortex) [reply build.gradle]: fixed

**Comment by gavv** [comment]
LGTM.

**Pull request merged**